robottelo/ui/contentviews.py

```diff
@@ -30,6 +30,21 @@ def go_to_filter_page(self, cv_name, filter_name):
         strategy, value = locators['contentviews.select_filter_name']
         self.click((strategy, value % filter_name))
 
+    def set_calendar_date_value(self, name, value=None):
+        """Set the input value of a date field and press the button to hide
+         the calendar popup panel"""
+        date_element = self.wait_until_element(
+            locators['contentviews.calendar_date_input'] % name)
+        date_element.clear()
+        if value:
+            date_element.send_keys(value)
```

Comment:
So if value was passed, we're setting it, if not (meaning user don't want to touch it) - we're clearing existing value and overwriting it with empty value, which is probably not something user will be happy about.
I'd rather
a) clear `date_element` only in case we're going to change it
b) changed `if value:` to `if value is not None:`. This way if user wants to clear the value, he'll be able to simply pass empty string `'' ` as value.